src/store/modules/AccountStore.js

```diff
@@ -33,28 +33,27 @@ const actions = {
         state
     }, payload) {
         return new Promise((resolve, reject) => {
-            let index = -1;
-            for (let i = 0; i < state.accountlist.length; i++) {
-                if (payload.account.chain == state.accountlist[i].chain && payload.account.accountID == state.accountlist[i].accountID) {
-                    reject('Account already exists');
+            let existingAccount = state.accountlist.find(x => x.chain == payload.account.chain && x.accountID == payload.account.accountID)
```

Comment:
What is this change about?

Reply:
IIRC the reject wasn't triggering within the loop so I removed the loop, kept the logic and now the account adding bug is gone 👍

Reply:
Ok, so its a change that is unrelated to this PR